types/src/genesis/mod.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::SectorSize;
+use address::Address;
+use num_bigint::bigint_ser;
+use serde::Serialize;
+use vm::TokenAmount;
+
+#[derive(Serialize)]
```

Comment:
Optionally just add a TODO that this won't serialize and deserialize as a string (default enum serialization is as a map)

Reply:
hmm, but my intention was for this enum to serialize/deserialize as a string.
I'm happy to resolve this TODO now, are you saying I need to changing something?

Reply:
Yeah, you can do so by manually implementing Serialize (and Deserialize if you intend to) and just serialize it how you want. I don't think there is anything you can tag an enum to achieve this specifically. 

for ref: https://serde.rs/enum-representations.html

Reply:
hmm, testing now and it seems to serialize fine as a string: 

```
{
  "Accounts": [],
  "Miners": [],
  "NetworkName": "localnet-bc9243f8-5ea9-4bd0-a96c-9eb471ac3872",
  "ActorType": "Account" // <- THIS
}
```

(I temporarily changed `Template` struct for testing)

```
#[derive(Serialize)]
#[serde(rename_all = "PascalCase")]
pub struct Template {
    pub accounts: Vec<Actor>,
    pub miners: Vec<Miner>,
    pub network_name: String,
    actor_type: ActorType,
    // timestamp: SystemTime,
}

impl Template {
    pub fn new(network_name: String) -> Template {
        Template {
            accounts: Vec::new(),
            miners: Vec::new(),
            network_name,
            actor_type: ActorType::Account,
        }
    }
}
```

Reply:
oh, my bad then, I didn't expect that to happen for cases with no data. Perhaps with this case you should make it all lower case to match their format as this is their strings:
```
	TAccount  ActorType = "account"
	TMultisig ActorType = "multisig"
```